.github/CODEOWNERS

```diff
@@ -1,56 +1,55 @@
 *                       @DataDog/agent-integrations
 /*/README.md            @DataDog/documentation
-/aerospike/             r.guo@aerospike.com
```

Comment:
If there is no gh user I think is better to leave the email than to delete the entry

Reply:
aerospike was moved to integrations-core so I removed the entry  https://github.com/DataDog/integrations-extras/pull/492

Reply:
ohh 🤦‍♀